starfish/image/_stack.py

```diff
@@ -960,3 +961,91 @@ def from_numpy_array(cls, array: np.ndarray) -> "ImageStack":
                     empty.set_slice({Indices.HYB: h, Indices.CH: c, Indices.Z: z}, view)
 
         return empty
+
+    def to_pixel_intensities(self, crop: Tuple[int, int, int]=(0, 0, 0)) -> IntensityTable:
+        """Generate an IntensityTable from all the pixels in the ImageStack
+
+        Parameters
+        ----------
+        crop : Tuple[int, int, int]
+            number of pixels from the image borders (z, y, x) to ignore
+
+        Returns
+        -------
+
+        """
+        zmin, ymin, xmin = crop
+        zmax = self.shape['z'] - zmin
```

Comment:
Do you want to use Indices.Z?

Reply:
I also think this should be `zmax = self.shape['z'] - crop`

Reply:
crop is a 3-tuple, so this doesn't work. I'm going to try this: 

```python
        crop_z, crop_y, crop_x = crop

        assert crop_z > 2 * self.shape[Coordinates.Z]
        assert crop_y > 2 * self.shape[Coordinates.Y]
        assert crop_x > 2 * self.shape[Coordinates.X]

        zmin = crop_z
        ymin = crop_y
        xmin = crop_x
        zmax = self.shape['z'] - crop_z
        ymax = self.shape['y'] - crop_y
        xmax = self.shape['x'] - crop_x
```